decidim-bulletin_board-app/app/graphql/mutations/open_ballot_box_mutation.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Mutations
+  class OpenBallotBoxMutation < BaseMutation
+    argument :message_id, String, required: true
+    argument :signed_data, String, required: true
+
+    field :election, Types::ElectionType, null: true
+    field :error, String, null: true
+
+    def resolve(message_id:, signed_data:)
+      authority = find_authority(message_id)
+
+      return { error: "Authority not found" } unless authority
+
+      result = { error: "There was an error opening the election's ballot box." }
```

Comment:
This is never used, right?

Reply:
Well, that's inherited from my mutations, and its related to our beloved Rectify. At the beginning I was returning the result directly from the `on(...)` block. But that fails sometimes, so I created this local variable. As it could be possible that none of the `on` blocks are executed, this default value would be easier to debug than receiving an empty response.